install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -112,8 +116,8 @@ function uninstall_containerd_artefacts() {
 
 function uninstall_nydus_snapshotter_artefacts() {
 	if host_systemctl list-units | grep -q nydus-snapshotter; then
-		for i in `ctr -n k8s.io snapshot --snapshotter nydus list | grep -v KEY | cut -d' ' -f1`; do
-			ctr -n k8s.io snapshot --snapshotter nydus rm $i || true
+		for i in `host_ctr -n k8s.io snapshot --snapshotter nydus list | grep -v KEY | cut -d' ' -f1`; do
+			host_ctr -n k8s.io snapshot --snapshotter nydus rm $i || true
```

Comment:
@fidencio I was about to ask you if there is a situation where `ctr` is not installed, but this `|| true` will handle any error.